### ISSUE TITLE
Add skip options for questions, one doesn't really understand

### ISSUE THREAD

**Issue by porky11** [8values/8values.github.io#63]
When I don't really understand, what a question means, or am not sure, or it doesn't matter at all for me, I mostly select neutral. There should be the option to skip something, so these questions won't be taken into account, and the end result is less neutral

**Comment by ninjaofbacon** [comment]
That's what the "neutral/unsure" button is for

**Comment by hsoj95** [comment]
I agree with @ninjaofbacon on this. The neutral/unsure button basically serves this purpose. In fact, implementing a "skip" button would in no wise change how the results calculate than the neutral/unsure button does currently.

**Comment by TristanBomb** [comment]
I think what porky11 means is that in the current system, answering Neutral moves you closer to the center compared to if the question simply didn't exist. The reason is because the total *possible* score for each axis is summed up and turned into a denominator for your results; answering neutral adds to the denominator\* but not the numerator, hence why you move towards neutral. This is intended behavior, but I can see why a change might be a good idea. I'm still on the fence though.

\* The denominator is actually summed before the quiz is taken, not after each question, but the point still stands.

**Comment by porky11** [comment]
I just know something like that from other tests, like the one for the german elections (Wahl-O-Mat), where skipped questions are not taken into account for the procentual match. Also you can set, how important something there is. Instead it may also be useful, just to be able to set different weights to the specific questions.

**Comment by ten7ei** [comment]
I agree. Someone can be neutral meaning just quite close to the center while having a skip or unsure button would remove the questions completely from the calculation which can still give less centered results.
At the same time it would be nice to give a notification to people which chose unsure/skip to tell the problem to this forum.
